cranelift/filetests/filetests/runtests/simd-vhighbits.clif

```diff
@@ -0,0 +1,53 @@
+test interpret
+test run
+target aarch64
+set enable_simd
+target x86_64
```

Comment:
This probably also needs a `machinst` so that it doesn't run on the legacy backend.